src/components/backpack/backpack.css

```diff
@@ -52,4 +52,5 @@
 .backpack-item {
     min-width: 4rem;
     margin: 0 0.25rem;
+    mix-blend-mode: multiply; /* Make white transparent for thumnbnails */
```

Comment:
Whoa...

Reply:
Would that break sprites that have white parts?

Reply:
Yea....